src/renderer/components/list/list.tsx

```diff
@@ -4,47 +4,76 @@
  */
 
 import styles from "./list.module.scss";
-import React, { useState } from "react";
-import { SearchInput } from "../input";
+import themeStyles from "./table-theme.module.scss";
 
-import type { UseTableOptions } from "react-table";
-import { ReactTable } from "../table/react-table";
+import React, { useEffect, useState } from "react";
+import { SearchInputUrl } from "../input";
+import type { TableOptions } from '@tanstack/react-table'
```

Comment:
3rd party lib is coupled with "List" but it should be only coupled with "Table".